### ISSUE TITLE
New youtube layout messing up all other layouts

### ISSUE THREAD

**Issue by lovebug0706** [lightbeam24/CustomTube#215]
I dont know if its just my browser (im using Firefox 115 ESR) but theres a new youtube layout messing up all of the other layouts. This includes all from 2008 - 2021. I hope this is just a bug on my end and not a new layout, but at least for the 2008 and 2009 layouts, the description is back on the right, which might make things easier.
![New Layout](https://github.com/lightbeam24/CustomTube/assets/34926216/17ab8662-be12-4fab-8f0a-1bbc60a125f6)
New Layout, as of April 10th 2024
![2012 Theme](https://github.com/lightbeam24/CustomTube/assets/34926216/394452af-79d0-4770-8f19-d72fff3d011d)
2012 Cosmic Panda theme
![2014 Theme](https://github.com/lightbeam24/CustomTube/assets/34926216/f871cb86-a97f-4f69-b2b0-d3df1ce5a429)
2014 Theme
![2008 Theme](https://github.com/lightbeam24/CustomTube/assets/34926216/203e6f14-63d8-4ce4-9b5d-225e843d6d4f)
2008 Theme

I hope theres a way we can fix this, because this is incredibly inconvenient. Thank you once again YouTube for messing up our layouts!

**Comment by MesaBlack** [comment]
EVen without the extension, mine's actually fine and not... that!

**Comment by gameshowrecaps** [comment]
You should probably use the CustomTube Beta extension, I believe it fixes all that.

**Comment by mjishnu** [comment]
> You should probably use the CustomTube Beta extension, I believe it fixes all that.

how do i use it ?

**Comment by MysticAx0lotl** [comment]
> You should probably use the CustomTube Beta extension, I believe it fixes all that.

Is it available on Firefox?

**Comment by gameshowrecaps** [comment]
> > You should probably use the CustomTube Beta extension, I believe it fixes all that.
> 
> Is it available on Firefox?

Yes, and follow the instructions here: https://github.com/lightbeam24/CustomTube/releases/tag/Beta

**Comment by mjishnu** [comment]
<img width="1280" alt="image" src="https://github.com/lightbeam24/CustomTube/assets/83004520/fa37d351-20c4-4e65-a8d4-6c293aa4c6d6">

it works but there is a slight bug i have a high dpi display i am running at 1440p so the video panel is not extend to the entire screen when i use zoom it make the text also become bigger. but thanks anyway at least its better than the current youtube default layout
